tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -35,6 +35,9 @@ spec:
     description: "aws eks enpoint to create clusters against"
   - name: cluster-name
     description: The name of the EKS cluster you want to spin.
+  results:
+    - name: datapoint
+      description: Stores the cl2 loadtest result to be consumed by cloudwatch  
```

Comment:
Consuming it is optional and any task besides CW can consume it. So I'd suggest the following wording:
`Stores the CL2 result that can be consumed by other tasks (e.g. cloudwatch)`